sky/cli.py

```diff
@@ -1288,6 +1288,11 @@ def status(all: bool, refresh: bool):  # pylint: disable=redefined-builtin
     after minutes of idling (no jobs running). If the time is followed by
     '(down)', e.g. '1m (down)', the cluster will be autodowned, rather than
     autostopped.
+
+    The estimated cost column indicates the price for the cluster based on the
+    type of resources being used and the duration of use up until the call
+    to status. This means if the cluster is UP, successive calls to status
+    will show increasing price.
```

Comment:
Probably add a note here: `The estimated cost is calculated based on the local cache of the cluster status, and may not be accurate for the cluster with autostop/use_spot set or terminated/stopped on the cloud console.`